evap/staff/templates/staff_course_type_index.html

```diff
@@ -2,26 +2,26 @@
 
 {% block breadcrumb %}
     {{ block.super }}
-    <li>{% trans "Course types" %}</li>
+    <li class="breadcrumb-item">{% trans "Course types" %}</li>
 {% endblock %}
 
 {% block content %}
     {{ block.super }}
 
-    <div>
-        <a href="{% url "staff:course_type_merge_selection" %}" class="btn btn-sm btn-default">{% trans "Merge course types" %}</a>
+    <div class="mb-3">
+        <a href="{% url "staff:course_type_merge_selection" %}" class="btn btn-sm btn-secondary">{% trans "Merge course types" %}</a>
     </div>
 
     <form id="course-type-form" method="POST" enctype="multipart/form-data" class="form-horizontal">
         {% csrf_token %}
         {{ formset.management_form }}
 
-        <table id="course_types_table" class="table">
+        <table id="course_types_table" class="table vertically-aligned">
             <thead>
                 <tr>
-                    <th>{% trans "Name (German)" %}</th>
-                    <th>{% trans "Name (English)" %}</th>
-                    <th>{% trans "Actions" %}</th>
+                    <th style="width: 45%">{% trans 'Name (German)' %}</th>
+                    <th style="width: 45%">{% trans 'Name (English)' %}</th>
+                    <th style="width: 10%">{% trans 'Actions' %}</th>
```

Comment:
usually both are okay. in some places i replaced the double quotes with single ones where the syntax highlighting didn't work correctly because there were double quotes within double quotes (e.g. when defining `{% trans "..." %}` in a `title="..."). after a while i started to use single quotes for all django template parts when i was changing something around them anyways - but it's still not consistent.